ee/ucp/deploy-application-package.md

```diff
@@ -8,7 +8,7 @@ keywords: ucp, swarm, kubernetes, application, app package
 
 Docker Enterprise Edition 2.1 introduces application packages in Docker. With application packages, you can add metadata and settings to an existing Compose file. This gives operators more context about applications they deploy and manage.
 
-An application package can have one of these formats:
+An application package can have one of the following formats:
```

Comment:
Change to read, "Application packages can present in one of two different formats, **Directory* or **Single-file**.

Reply:
Can you clarify this comment? Should

Reply:
My apologies, @traci-morrison. I was going down one path in my edit, changed tracks, and then failed to clean up the half-baked suggestion that I was first making.

I suggest that the leading phrase be entirely deleted in favor of "Application packages can present in one of two different formats, Directory or Single-file." That alone will suffice for leading into the bullet descriptions for the two formats.

Reply:
Thanks, I made that change. PTAL.